.github/workflows/l10n_extract.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
         run: |
           cd fxa-code
           yarn workspaces focus fxa-content-server fxa-auth-server fxa-payments-server fxa-settings
-          yarn workspace fxa-settings build
+          yarn workspace fxa-settings grunt merge-ftl
```

Comment:
@LZoog 

So from local testing, it appears the concat file appears in the `en-US` locale when built with `grunt merge-ftl`, but currently fxa-content-server-l10n uses the en locale for fxa-settings. I'm having trouble getting `yarn workspace fxa-settings build` to work locally so I can't confirm my hypothesis but my assumption is that fxa-settings used to only generate the `en` locale. 

I think this [line](https://github.com/mozilla/fxa/blob/befccf53df44eebe3422a384a8a5b8930f8bcad8/packages/fxa-settings/Gruntfile.js#L14) might need to be adjusted to output to en. 

I confirmed that fxa-auth-server builds into the `en` locale.

Reply:
Looking at the script in the l10n repository, I think we expect the file to be generated in en-US?
https://github.com/mozilla/fxa-content-server-l10n/blob/main/scripts/extract_strings.sh#L121

Reply:
😓 
There’s the step I missed. Apologies, ignore me.

Reply:
We do currently expect it to be generated in `en-US`. We have [FXA-6003](https://mozilla-hub.atlassian.net/browse/FXA-6003) filed and reference that line^ in this repo to change in that ticket. 🙂 I appreciate the thoroughness.